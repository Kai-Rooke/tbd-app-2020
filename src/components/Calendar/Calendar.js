import React from 'react';
import './Calendar.css';
import UCPButton from './UCPButton';
import DropdownMenu from './DropdownMenu';
import {Container} from 'react-bootstrap';

const options = [
  { value: '0', label: '9:00' },
  { value: '1', label: '10:00'},
  { value: '2', label: '11:00'},
  { value: '3', label: '12:00'},
  { value: '4', label: '13:00' },
  { value: '5', label: '14:00'},
  { value: '6', label: '15:00'},
  { value: '7', label: '16:00'},
]
export default class Calendar extends React.Component{
  
 render(){
     return (

     

      <Container>
      <DropdownMenu options/>
      <table>
     
        <th><UCPButton className="mediumbutton" buttonText="Monday"/></th>
        <th><UCPButton className="mediumbutton" buttonText="Tuesday"/></th>
        <th><UCPButton className="mediumbutton" buttonText="Wednesday"/></th>
        <th><UCPButton className="mediumbutton" buttonText="Thursday"/></th>
        <th><UCPButton className="mediumbutton" buttonText="Friday"/></th>
      
      </table>
    </Container>
     )
 }
};