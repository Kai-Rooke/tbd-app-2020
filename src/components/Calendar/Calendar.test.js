import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import Calendar from './Calendar';

//cleans up 
afterEach(cleanup);

it("renders", () => {
    const { asFragment } = render(<Calendar/>);
    expect(asFragment()).toMatchSnapshot();
});
