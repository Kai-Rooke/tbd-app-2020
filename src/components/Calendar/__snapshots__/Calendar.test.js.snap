// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="WeeksButton row"
    >
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="largebutton"
            data-testid="ucpButton"
            type="button"
          >
            Week Commencing 26/10/2020​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="largebutton"
            data-testid="ucpButton"
            type="button"
          >
            Week Commencing 2/11/2020​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="largebutton"
            data-testid="ucpButton"
            type="button"
          >
            Week Commencing 9/11/2020​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="largebutton"
            data-testid="ucpButton"
            type="button"
          >
            Week Commencing 16/11/2020​
          </button>
        </a>
      </div>
    </div>
    <br />
    <br />
    <div
      class="DaysButton row"
    >
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            type="button"
          >
            Monday​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            type="button"
          >
            Tuesday​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            type="button"
          >
            Wednesday​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            type="button"
          >
            Thursday​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            type="button"
          >
            Friday​
          </button>
        </a>
      </div>
    </div>
    <br />
    <br />
    <div
      class="row"
    >
      <div
        class="col"
      >
        <div
          class="Dropdown css-2b097c-container"
        >
          <div
            class=" css-yk16xz-control"
          >
            <div
              class=" css-g1d714-ValueContainer"
            >
              <div
                class=" css-1wa3eu0-placeholder"
              >
                Select...
              </div>
              <div
                class="css-b8ldur-Input"
              >
                <div
                  class=""
                  style="display: inline-block;"
                >
                  <input
                    aria-autocomplete="list"
                    autocapitalize="none"
                    autocomplete="off"
                    autocorrect="off"
                    id="react-select-2-input"
                    spellcheck="false"
                    style="box-sizing: content-box; width: 2px; border: 0px; font-size: inherit; opacity: 1; outline: 0; padding: 0px;"
                    tabindex="0"
                    type="text"
                    value=""
                  />
                  <div
                    style="position: absolute; top: 0px; left: 0px; visibility: hidden; height: 0px; overflow: scroll; white-space: pre; font-size: inherit; font-family: -webkit-small-control; letter-spacing: normal; text-transform: none;"
                  />
                </div>
              </div>
            </div>
            <div
              class=" css-1hb7zxy-IndicatorsContainer"
            >
              <span
                class=" css-1okebmr-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                class=" css-tlfecz-indicatorContainer"
              >
                <svg
                  aria-hidden="true"
                  class="css-6q0nyr-Svg"
                  focusable="false"
                  height="20"
                  viewBox="0 0 20 20"
                  width="20"
                >
                  <path
                    d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="col"
      >
        <div
          class="jumbotron"
        >
          <br />
          <div
            class="jumbotron"
            data-testid="jumbotroncomponent"
          >
            <p>
              <b>
                PRIVACY STATEMENT:
              </b>
              <br />
               We will only use your personal information to administer your application and / or enrolment and to provide the products and services you have requested from us.
              <br />
               Further information on our privacy policy can be found at https://www.ucp.ac.uk/privacy-and-cookies/ or by emailing 
              <a
                class="text"
                href="mailto:hello@ucp.ac.uk"
              >
                hello@ucp.ac.uk
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="DaysButton row"
    >
      <br />
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            type="button"
          >
            Refresh​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <a
          href="/emptyLink"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            type="button"
          >
            Confirm​
          </button>
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
